src/components/Table.js

```diff
@@ -1,10 +1,17 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import './Table.css';
+import getDistanceFromLatLonInKm from '../lib/haversineFunction';
```

Comment:
Hmm. Seeing the code here that’s duplicated in `Map.js`, I wonder if some of this could be moved to a common location? Not a big deal since it’s working, but if we had more time might be a good refactor to make these views more “dumb” (i.e. just taking in data and rendering it without doing any transformation)!

Reply:
definitely could use a refactor!